docs/workflow_solutions/creating_sandbox_apps.md

```diff
@@ -0,0 +1,180 @@
+# Open OnDemand
+
+[Open OnDemand](https://rc.uab.edu)(OOD) is the web interface to Cheaha HPC resources. We have setup multiple general purpose apps, such as HPC Desktop, Jupyter, RStudio etc, on Cheaha, to make the process of accessing these basic softwares easier. OOD also allows users to setup their own personal applications, as Sandbox apps.
+
+This article will walk you through how to set up an OOD Sandbox app. For ease of understanding, the article is split into the following sections:
+
+1. [Setting up OOD Sandbox for your Cheaha Account](#setting-up-ood-sandbox-for-your-cheaha-account)
+1. [Building your first dev app](#building-your-first-dev-app)
+   - [FSL](#fsl)
```

Comment:
Maybe introduce FSL with "Tutorial: [FSL]...". At least until we have more than one.